### ISSUE TITLE
Generated code encountered errors with explicitly parameter name for method

### ISSUE THREAD

**Issue by ecoopnet** [Brightify/Cuckoo#199]
While Swift language 4.1 allows this code, 
Cuckoo fails to generate mock/stub for FooImpl class.

```
import Foundation

protocol Foo {
    func doSomething(for: String)
}

class FooImpl: Foo {
    func doSomething(for value: String) {
        print("something was done for \(value)")
    }

}
```

I found the reason of the failure.

It fails always when:
- A method in protocol only have argument label **without** parameter name (i.e. `func doSomething(for: String)` )
- the method implementation in class have argument label **with** parameter name (i.e. `func doSomething(for value: String)` )

https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Declarations.html#//apple_ref/swift/grammar/function-declaration 

Sample Project: https://github.com/ecoopnet/cuckoo-bug-sample

### Environment:
- Cuckoo: 0.11.3 (latest in CocoaPods)
- Xcode: 9.3(9E145)
- macOS: High Sierra

**Comment by ecoopnet** [comment]
I was finally changed my code like below to avoid this problem for now.
```
import Foundation

protocol Foo {
    func doSomething(for value: String)
}

class FooImpl: Foo {
    func doSomething(for value: String) {
        print("something was done for \(value)")
    }

}
```

**Comment by TadeasKriz** [comment]
@ecoopnet Thank you for this perfect report. We'll look into it.

**Comment by ecoopnet** [comment]
🙇

**Comment by MatyasKriz** [comment]
Hey, I'm sorry, I didn't really notice your issue before this. It was a pretty quick fix, so I hope you didn't suffer too much because of this bug.

It's fixed by PR #291 and released in `1.0.6`, because it seems to me like this could affect more users.
